spot-ml/src/test/scala/org/apache/spot/SpotSparkLDAWrapperTest.scala

```diff
@@ -0,0 +1,11 @@
+package org.apache.spot
+

```

Comment:
it feels a little off to check in an empty test... maybe add a simple smoke test